src/System.AppContext/src/project.json

```diff
@@ -1,9 +1,12 @@
 {
   "frameworks": {
-    "dnxcore50": {
+    "netstandard1.5": {
       "dependencies": {
-        "Microsoft.TargetingPack.Private.CoreCLR": "1.0.0-rc3-23910"
-      }
+        "Microsoft.TargetingPack.Private.CoreCLR": "1.0.0-rc3-23918"
+      },
+      "imports": [
+        "dotnet5.6"
```

Comment:
This seems odd I wouldn't expect a need for a dotnet5.6 import for the coreclr targeting pack which only has a netstandard1.0 lib in it.